recommenders/evaluation/spark_evaluation.py

```diff
@@ -356,6 +358,72 @@ def map_at_k(self):
         """
         return self._metrics.meanAveragePrecisionAt(self.k)
 
+    def r_precision(self):
```

Comment:
could you please separate this into the other PR https://github.com/recommenders-team/recommenders/pull/2219